runtime-modules/content/src/tests/nft/make_bid.rs

```diff
@@ -807,7 +808,7 @@ fn make_bid_ok_with_english_auction_completion_and_total_balance_slashed() {
         ));
 
         assert_eq!(
-            Balances::<Test>::free_balance(&SECOND_MEMBER_ACCOUNT_ID),
+            Balances::<Test>::usable_balance(&SECOND_MEMBER_ACCOUNT_ID),
```

Comment:
I think in this case it would actually be best to use:
```suggestion
            Balances::<Test>::total_balance(&SECOND_MEMBER_ACCOUNT_ID),
```
Because:
- It's a better way to make sure that the funds are actually slashed (not just reserved, locked etc.)
- It's inline with the test function name (make_bid_ok_with_english_auction_completion_and_**total_balance**_slashed)

The same applies to _make_bid_ok_with_open_auction_completion_and_total_balance_slashed_